src/components/DisplayObject.js

```diff
@@ -0,0 +1,133 @@
+import invariant from 'fbjs/lib/invariant'
+import { DisplayObject } from 'pixi.js'
+import { PROPS_DISPLAY_OBJECT, PROPS_RESERVED } from '../utils/props'
+import { isPointType, parsePoint } from '../utils/pixi'
+import { not, hasKey } from '../helpers'
+
+DisplayObject.getReactEventHandlers = function() {
```

Comment:
Can we perhaps use `PIXI.DisplayObject.mixin` instead to monkey patch the DisplayObject? Not sure, haven't used it.

Reply:
Using `mixin` then we will definitely destroy compatibility with v4.

Reply:
You're right, it's just syntax sugar:

https://github.com/pixijs/pixi.js/blob/45df7ebaa9110743606440ffef5a24432ea66961/packages/display/src/DisplayObject.ts#L58-L78